Mathlib/Analysis/Fourier/Char.lean

```diff
@@ -0,0 +1,190 @@
+/-
```

Comment:
I'd rename the file to BoundedContinuousFunctionChar, because the point of the current file is to work with a character bundled as a continuous bounded function, not a general character.